src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
+import React from 'react';
 // export const Person = ({ person }) => ();
+export function Person({ person }) {
+  function Text() {
+    if (person.sex === 'm' && person.isMarried) {
+      return `${person.partnerName} is my wife`;
+    }
+
+    if (!person.isMarried) {
+      return `Im not married`;
+    }
+
+    if (person.sex === 'f' && person.isMarried) {
+      return `${person.partnerName} is my husband`;
+    }
```

Comment:
Combine similar checks
```suggestion
    if (person.isMarried) {
      return  `${person.partnerName} is my ${person.sex === 'm'  ? 'wife' : 'husband'}`;
    } else {
      return `Im not married`;
    }
```